vnext/ReactUWP/Views/TextInputViewManager.cpp

```diff
@@ -110,28 +121,65 @@ void TextInputShadowNode::createView() {
   //
   // TextChanging is used to drop the Javascript response of 'A' and expect
   // another TextChanged event with correct event count.
-  m_textBoxTextChangingRevoker = textBox.TextChanging(
-      winrt::auto_revoke, [=](auto &&, auto &&) { m_nativeEventCount++; });
+  if (m_isTextBox) {
+    m_textBoxTextChangingRevoker = control.as<winrt::TextBox>().TextChanging(
```

Comment:
>m_textBoxTextChangingRevoker [](start = 4, length = 28)

Since you supports dynamitic change, when m_textBoxTextChangingRevoker is assigned, m_passwordBoxPasswordChangingRevoker should be set to null explicit.
The same rule applies to other revoker